hardhat.config.ts

```diff
@@ -2,10 +2,19 @@ import { HardhatUserConfig } from "hardhat/types";
 import "@shardlabs/starknet-hardhat-plugin";
 import "@nomiclabs/hardhat-ethers";
 import chai from "chai";
-import {solidity} from "ethereum-waffle";
+import { solidity } from "ethereum-waffle";
+import { config as dotenvConfig } from "dotenv";
+import { resolve } from "path";
+
+dotenvConfig({ path: resolve(__dirname, "./.env") });
```

Comment:
is this line needed? when the env variable file is `env`, we don't need to add it

Reply:
yes, it's needed, it will fail to read the variables from .env without it!